### PR TITLE
qemu.tests.cfg.usb.cfg: fix usb controller speed mismatch problem

### PR DESCRIPTION
**Pull request by hereischen**
added 'no usb-ehci' into usb-hub and usb-ccid related variants to drop
these cases.

Signed-off-by: Haotong Chen <hachen@redhat.com>

ID: 1441539

### CODE REVIEW

**Comment by CongLi** [comment]
The problem is only for RHEL7 host according to your test result.
Please confirm with RHEL6 host too.

**Comment by yanan-fu** [comment]
I remember, it is better to add "qemu.test.cfg"  in the title, thanks!

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by hereischen** [comment]
@CongLi 
usb-ehci has speed mismatch issue with usb-hub and usb-ccid for RHEL7.
For REHL6, usb-echi has this issue with usb-hub, usb-kbd, usb-mouse, usb-tablet and usb-ccid

**Comment by hereischen** [comment]
@yanan-fu edited, thanks

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Comment by yanan-fu** [comment]
LGTM 
Acked-by: Yanan Fu yfu@redhat.com

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -85,6 +86,8 @@
         - @usb_nodev:
             only usb_storage, usb_host, usb_multi_disk
         - usb_kbd:
+            Host_RHEL.m6:
+               no usb-ehci
```
Comment (CongLi) [inline qemu/tests/cfg/usb.cfg]: 3 blanks ?

**Comment by CongLi** [comment]
Ack.

**Comment by CongLi** [comment]
Thanks all.

**Comment by CongLi** [comment]
Hi @hereischen 

It's better to update commit log too which is more clear.
like
fix usb speed mismatch problem.
host rhel7: ***
host  rhel6: ***

**Comment by CongLi** [comment]
Ack.

**Pull request merged**